mso/resource_mso_schema_site_vrf_region.go

```diff
@@ -157,7 +163,7 @@ func resourceMSOSchemaSiteVrfRegionImport(d *schema.ResourceData, m interface{})
 	stateVrf := get_attribute[4]
 	stateRegion := get_attribute[6]
 
-	for i := 0; i < count; i++ {
+	for i := 0; i < count && !found; i++ {
```

Comment:
It would be good to include the template name as well in the import statement.

terraform import mso_schema_site_vrf_region.vrfRegion {schema_id}/site/{site_id}-{template_name}/vrf/{vrf_name}/region/{region_name}

After that we can retrieve the name of the template from the import statement through string manipulation and use it as part of the condition -> if apiSite == stateSite && apiTemplate == stateTemplate { (same as Read)

Reply:
ok